sources/osg/Node.js

```diff
@@ -227,29 +227,39 @@ define( [
             return this.boundingSphere;
         },
 
-        computeBound: function ( bsphere ) {
+        // TODO: PERF: Heavy GC impact
+        // find why making a closure var for bb breaks (v8&&ffx)
+        // frustum culling sample
+        //var  bbTemp = new BoundingBox();
```

Comment:
Tried, but had a set/copy problem with "center", could try again now that you can provide a result to bbox.center()